also xfail test_maxiter_worsening for Python 3.10 on aarch64
see also https://github.com/scipy/scipy/issues/13019 + https://github.com/scipy/scipy/pull/13022
author: Kenneth Hoste (HPC-UGent)
diff -ru scipy-1.10.1.orig/scipy/sparse/linalg/_isolve/tests/test_iterative.py scipy-1.10.1/scipy/sparse/linalg/_isolve/tests/test_iterative.py
--- scipy-1.10.1.orig/scipy/sparse/linalg/_isolve/tests/test_iterative.py	1970-01-01 00:00:00.000000000 +0000
+++ scipy-1.10.1/scipy/sparse/linalg/_isolve/tests/test_iterative.py	2023-02-25 17:18:12.432151563 +0000
@@ -451,7 +451,7 @@
 
 @pytest.mark.parametrize("solver", [
     pytest.param(gmres, marks=pytest.mark.xfail(platform.machine() == 'aarch64'
-                                                and sys.version_info[1] == 9,
+                                                and sys.version_info[1] >= 9,
                                                 reason="gh-13019")),
     qmr,
     pytest.param(lgmres, marks=pytest.mark.xfail(
